common/dynamicconfig/constants.go

```diff
@@ -849,6 +849,11 @@ of Timeout and if no activity is seen even after that the connection is closed.`
 		true,
 		`FrontendEnableSchedules enables schedule-related RPCs in the frontend`,
 	)
+	FrontendMaxFairnessWeightOverrideConfigLimit = NewGlobalIntSetting(
```

Comment:
We don't need another config for this, just use the max overrides. We're never going to configure it separately.